cmd/node/flags.go

```diff
@@ -507,26 +517,66 @@ func getWorkingDir(workingDir string, log logger.Logger) string {
 }
 
 func applyCompatibleConfigs(log logger.Logger, configs *config.Configs) error {
-	importDbFlags := configs.ImportDbConfig
-	importDbFlags.ImportDbNoSigCheckFlag = importDbFlags.ImportDbNoSigCheckFlag && importDbFlags.IsImportDBMode
-	importDbFlags.ImportDbSaveTrieEpochRootHash = importDbFlags.ImportDbSaveTrieEpochRootHash && importDbFlags.IsImportDBMode
+	if configs.FlagsConfig.EnablePprof {
+		runtime.SetMutexProfileFraction(5)
```

Comment:
what is this?

Reply:
it is the setting for allowing the mutex profiling. It's only set when the profile mode is enabled.
It's old code, just moved

Reply:
right, did not see that you've moved the code